Sources/Ledger/TxOutUtils.swift

```diff
@@ -101,6 +99,35 @@ enum TxOutUtils {
         }
     }
 
+    static func calculateCrc32(
```

Comment:
Moved this call back to TxOutUtils so it fits in better with other similar calls

Reply:
@the-real-adammork I don't think anything is using this any longer - since you can just do <data>.commitmentCrc32 now, anyplace that was using this is now using the commitmentCrc32 ... or did I miss something?

Reply:
scratch that - the impl is here, and Data32 calls it, instead of the impl being in Data32...